pkg/cmd/account/delete/delete.go

```diff
@@ -70,8 +66,8 @@ func NewCmdDelete(f factory.Factory) *cobra.Command {
 			return delete(client, itemToDelete)
 		},
 	}
-	// TODO confirm might want to be a global flag?
-	cmd.Flags().BoolP("confirm", "y", false, "Don't ask for confirmation before deleting the account.")
+
+	question.RegisterDeleteFlag(&skipConfirmation, cmd, "account")
```

Comment:
I feel like this would read better if `cmd` was first.

This is very subjective and opinion based. Imagine if it were C# and we were writing an extension method, it would more likely go on the cobra command than it would on the skipConfirmation variable. 
Take it or leave it 🤷